README.md

```diff
@@ -8,7 +8,59 @@ Queries for text objects are also included which help you to navigate, select, a
 
 ## Installation
 
-- If you want ReScript Tree-sitter in NeoVim, refer to [`nvim-treesitter-rescript`](https://github.com/nkrkv/nvim-tree-sitter-rescript/) installation notes;
+### Neovim
+
+If you want ReScript Tree-sitter in NeoVim, you will first need to register a new parser for it like so:
+
+```lua
+local parser_config = require("nvim-treesitter.parsers").get_parser_configs()
+parser_config.rescript = {
+  install_info = {
+    url = "https://github.com/nkrkv/tree-sitter-rescript",
+    branch = "main",
+    files = { "src/scanner.c" },
+    generate_requires_npm = false,
+    requires_generate_from_grammar = true,
+    use_makefile = true, -- macOS specific instruction
+  },
+}
+```
+
+This will make `TSInstall rescript` globally available. For more persistent approach you should add this parser to your Lua configuration.
+
+Default configuration detects `.res` and `.resi` files. You can confirm that it's correctly installed by using [`nvim-treesitter/playground`](https://github.com/nvim-treesitter/playground) and invoking `TSPlaygroundToggle` when you are in the ReScript file.
```

Comment:
Since Neovim 0.9 the command `:InspectTree` is available to inspect the tree.

Reply:
Will update it!